data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/metrics/Metrics.java

```diff
@@ -106,6 +106,12 @@ public class Metrics {
    */
   public static final String EVENT_DISPATCH_LATENCY = "event_dispatch_latencies";
 
+  /**
+   * @link https://knative.dev/docs/eventing/observability/metrics/eventing-metrics/
+   * @see Metrics#eventDispatchRetry(io.micrometer.core.instrument.Tags)
+   */
+  public static final String EVENT_DISPATCH_RETRY = "event_dispatch_retries";
```

Comment:
```suggestion
  public static final String EVENT_DISPATCH_RETRY = "event_dispatch_retries_count";
```

Reply:
Yes @matzew, we can name it as `event_dispatch_retry_count` @pierDipi WDYT?

Reply:
Sounds good

Reply:
Let's align the method name as well

Reply:
done